### PR TITLE
Added label file upload for text

### PR DESCRIPTION
**Pull request by cpranav93**
@laurasootes, I have added label file upload support to text as well. However, I am not able to test it due to WSL issues for text. Could you give it a go and let me know if it works fine?

### CODE REVIEW

File: dashboard/callbacks.py [hunk 1]
```diff
@@ -51,9 +50,9 @@
 cache.clear()
 
 # global variables # replace by generic label loader
-class_name_mnist = ['digit 0', 'digit 1']
-class_name_text = ["negative", "positive"]
-class_names_imagenet = [imagenet_class_name(idx) for idx in range(1000)]
+# class_name_mnist = ['digit 0', 'digit 1']
```
Comment (laurasootes) [inline dashboard/callbacks.py:53]: these lines can be removed instead of commented

Comment (laurasootes) [reply dashboard/callbacks.py:53]: additionally, I just noticed that there is a print statement in line 158 from the image label update

**Review by laurasootes — COMMENTED** [review]
I think it is not working yet, somehow it is using the second character (can even be a space) of the input string as prediction label:

image

image

File: dashboard/callbacks.py (hunk 1 above)
Comment (cpranav93) [reply dashboard/callbacks.py]: Comments and print statement removed.

**Comment by cpranav93** [comment]
The label issue should be fixed now. Could you give it a test? @laurasootes

File: dashboard/callbacks.py [hunk 2]
```diff
@@ -537,9 +559,10 @@ def update_multi_options_t(fn_m, input_text, sel_methods, new_model, new_text,
             onnx_model_path, word_vector_path, max_filter_size=5)
 
         try:
+            print ('label:', labelnames)
```
Comment (laurasootes) [inline dashboard/callbacks.py:562]: Can you remove this print statement

**Review by laurasootes — APPROVED** [review]
Everything works and looks good! If you can remove this one redundant print statement it is ready to go :)

**Pull request merged**